src/components/TimedOut.tsx

```diff
@@ -0,0 +1,9 @@
+import { Text } from '@mantine/core';
+
+export function TimedOut() {
+  return (
+    <Text>
+      Thank you for participating. Unfortunately you have didn&apos;t answer the questions in time and you are not eligible to participate in the study. You may close this window now.
```

Comment:
```suggestion
      Thank you for participating. Unfortunately, you have not answered the questions within the given time limit. Because of this, you are no longer eligible to participate in the study. You may close this window now.
```
(Or something along these lines. At least change the "you have didn't" haha)